vulnerabilities/models.py

```diff
@@ -202,11 +221,40 @@ def __str__(self):
     class Meta:
         unique_together = ("package", "vulnerability")
         verbose_name_plural = "PackageRelatedVulnerabilities"
+        indexes = [models.Index(fields=["fix"])]
 
-
-class ImportProblem(models.Model):
-
-    conflicting_model = models.JSONField()
+    def update_or_create(self):
+        """
+        Update if supplied record has more confidence than existing record
+        Create if doesn't exist
+        """
+        try:
+            existing = self.__class__.objects.get(
```

Comment:
Why the `__class__` here?
```suggestion
            existing = PackageRelatedVulnerability.objects.get(
```